frontend/cypress/e2e/account/forgot-password.cy.ts

```diff
@@ -0,0 +1,138 @@
+describe('Forgot password', () => {
```

Comment:
Tests sehen gut aus. Mir gefällt:
- Die mocks 
- die waits mit horchen 
- schöne struktur 

man könnte noch testen:
- The user enters an invalid email address.

aber pass so! Sehr gut!

Reply:
true